x-pack/plugin/runtime-fields/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+/* TODO this exists so that we can run painless in unit tests which we
```

Comment:
I think we can remove this. I'll add it to my list of things today.

Reply:
It also shouldn't block merging.

Reply:
thanks

Reply:
I pushed 34e576fd04cdcfa08fea1ddef3a65f34a142917d to drop it.